components/p3a/brave_p3a_service.cc

```diff
@@ -397,6 +397,9 @@ void BraveP3AService::StartScheduledUpload() {
   }
 
   // Only upload if service is enabled.
+#if defined(OS_ANDROID)
+  p3a_enabled = false;
+#endif  // OS_ANDROID
   bool p3a_enabled = local_state_->GetBoolean(brave::kP3AEnabled);
```

Comment:
This clobbers the value you've just set. Move your change down a line?

Reply:
whoops; definitely my bad there. Thanks!

Reply:
Fixed with https://github.com/brave/brave-core/pull/10940/commits/1333b23bdcf4ce07aa31bdf85e6a769716a5c6e3